gulpfile.js

```diff
@@ -1,111 +1,101 @@
-const gulp = require('gulp');
-const babel = require('gulp-babel');
-const browsersync = require("browser-sync").create();
-const concat = require('gulp-concat');
-const uglify = require('gulp-uglify');
-const sourcemaps = require('gulp-sourcemaps');
-const browserify = require('browserify');
-const babelify = require('babelify');
-const source = require('vinyl-source-stream');
-const buffer = require('vinyl-buffer');
+const gulp = require('gulp')
+const browsersync = require('browser-sync').create()
+const uglify = require('gulp-uglify')
+const sourcemaps = require('gulp-sourcemaps')
+const browserify = require('browserify')
+const babelify = require('babelify')
+const source = require('vinyl-source-stream')
+const buffer = require('vinyl-buffer')
 
 var paths = {
 	dev: {
 		html: 'demo/typopo-demo.html',
 		index: 'typopo-demo.html',
 		name: 'typopo_browser.built.js',
-		dest: 'build/'
+		dest: 'build/',
 	},
 	browser: {
 		src: 'src/browser_typopo.js',
 		name: 'typopo.min.js',
-		dest: 'dist/'
+		dest: 'dist/',
 	},
 	npm: {
 		src: 'src/typopo.js',
 		name: 'typopo_dist.min.js',
-		dest: 'dist/'
-	}
-};
+		dest: 'dist/',
+	},
+}
 
 /*
  * Define our tasks using plain functions
  */
 
 function devBrowserBuild() {
-	 return browserify({entries: paths.browser.src, debug: true})
-		 .transform("babelify")
-		 .bundle()
-		 .pipe(source(paths.dev.name))
-		 .pipe(buffer())
-		 .pipe(sourcemaps.init())
-		 .pipe(sourcemaps.write('./maps'))
-		 .pipe(gulp.dest(paths.dev.dest))
- };
-
+	return browserify({ entries: paths.browser.src, debug: true })
+		.transform('babelify')
+		.bundle()
+		.pipe(source(paths.dev.name))
+		.pipe(buffer())
+		.pipe(sourcemaps.init())
+		.pipe(sourcemaps.write('./maps'))
+		.pipe(gulp.dest(paths.dev.dest))
+}
 
 function browserBuild() {
 	return browserify({ entries: paths.browser.src, debug: false })
-	.transform(babelify)
-	.bundle()
-	.pipe(source(paths.browser.name))
-	.pipe(buffer())
-	.pipe(uglify())
-	.pipe(gulp.dest(paths.browser.dest));
+		.transform(babelify)
+		.bundle()
+		.pipe(source(paths.browser.name))
+		.pipe(buffer())
+		.pipe(uglify())
+		.pipe(gulp.dest(paths.browser.dest))
 }
 
 function npmBuild() {
-	return browserify({entries: paths.npm.src}, {standalone: 'typopo'})
-		.transform("babelify")
+	return browserify({ entries: paths.npm.src }, { standalone: 'typopo' })
+		.transform('babelify')
 		.bundle()
 		.pipe(source(paths.npm.name))
 		.pipe(buffer())
 		.pipe(uglify())
 		.pipe(gulp.dest(paths.npm.dest))
-};
+}
 
 function copyHtmlToDest() {
-	return gulp
-		.src(paths.dev.html)
-		.pipe(gulp.dest(paths.dev.dest));
-};
+	return gulp.src(paths.dev.html).pipe(gulp.dest(paths.dev.dest))
```

Comment:
How does prettier decide whether to stack functions or inline them? (When comparing to previous adjustments)

Reply:
Prettier is trying to stuff as much chained methods into a single line as possible – the "possible" is maximum chars in one line, defined by the `printWidth` option (80 by default). Overflowing that triggers wrapping every method to a new line. (https://prettier.io/docs/en/options.html#print-width)